utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/dawn.lua

```diff
@@ -0,0 +1,52 @@
+local ubus = require "ubus"
+
+local function scrape()
+  local metric_dawn_ap_channel_utilization_ratio = metric("dawn_ap_channel_utilization_ratio","gauge")
+  local metric_dawn_ap_stations_total = metric("dawn_ap_stations_total","gauge")
+  local metric_dawn_station_signal_dbm = metric("dawn_station_signal_dbm","gauge")
+
+  local u = ubus.connect()
+  local network = u:call("dawn", "get_network", {})
+
+  for ssid, ssid_table in pairs(network) do
+    for ap, ap_table in pairs(ssid_table) do
+
+      if (ap_table['local'] == true) then
+
+        local ht_support = (ap_table['ht_support'] == true) and 1 or 0
+        local vht_support = (ap_table['vht_support'] == true) and 1 or 0
+
+        local labels = {
+          ssid = ssid,
+          bssid = ap,
+          freq = ap_table['freq'],
+          ht_support = ht_support,
+          vht_support = vht_support,
+          neighbor_report = ap_table['neighbor_report'],
+        }
+        metric_dawn_ap_channel_utilization_ratio(labels, ap_table['channel_utilization'])
```

Comment:
Here `ap_table['channel_utilization']` is still 0..255, correct? For ratio it needs to be 0..1.

```suggestion
        metric_dawn_ap_channel_utilization_ratio(labels, ap_table['channel_utilization'] / 255)
```